### PR TITLE
Add new e2e-gcp-sdn workflow

### PR DESCRIPTION
**Pull request by jluhrsen**
Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

**Comment by jluhrsen** [comment]
a previous version of this PR contained a job change to get a rehearsal to verify this works as expected. it does.
[job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/30031/rehearse-30031-periodic-ci-openshift-release-master-ci-4.12-e2e-gcp-sdn/1542566792818331648)
[get pods to show openshift-sdn pods](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/30031/rehearse-30031-periodic-ci-openshift-release-master-ci-4.12-e2e-gcp-sdn/1542566792818331648/artifacts/e2e-gcp-sdn/gather-extra/artifacts/oc_cmds/pods)
[network config file output](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/30031/rehearse-30031-periodic-ci-openshift-release-master-ci-4.12-e2e-gcp-sdn/1542566792818331648/artifacts/e2e-gcp-sdn/sdn-conf/build-log.txt)

File: ci-operator/step-registry/openshift/e2e/gcp/sdn/openshift-e2e-gcp-sdn-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"path": "openshift/e2e/gcp/sdn/openshift-e2e-gcp-sdn-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"abhinavdahiya",
```
Comment (patrickdillon) [inline ci-operator/step-registry/openshift/e2e/gcp/sdn/openshift-e2e-gcp-sdn-workflow.metadata.json:5]: abhinav is not with rh anymore

**Review by patrickdillon — COMMENTED** [review]
/approve

**Pull request merged**